UnityProject/Assets/Scripts/Medical/CloningConsole.cs

```diff
@@ -92,6 +100,12 @@ public void Scan()
 			CreateRecord(mob, playerScript);
 			scanner.statusString = "Subject successfully scanned.";
 		}
+		else {
+			if(!scanner) scanner.statusString = "Scanner not found.";
```

Comment:
you could pull this out into a function, so you don't have to repeat the code 
something like 
private void UpdateStatus(){}

Reply:
This can work for checking scanner, apcconnected, and voltage. I'll make that now.